spring-xd-dirt/src/main/java/org/springframework/xd/dirt/plugins/spark/streaming/SparkStreamingMessageConverterSupport.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.xd.dirt.plugins.spark.streaming;
+
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.integration.channel.AbstractMessageChannel;
+import org.springframework.util.MimeType;
+import org.springframework.xd.dirt.plugins.stream.ModuleTypeConversionPlugin;
+import org.springframework.xd.dirt.plugins.stream.ModuleTypeConversionSupport;
+import org.springframework.xd.dirt.util.ConfigLocations;
+
+/**
+ * Utility class to support message conversion in spark streaming module.
+ *
+ * @author Ilayaperumal Gopinathan
+ */
+public class SparkStreamingMessageConverterSupport {
```

Comment:
Now that the method below has been factored out, I'm wondering if it would make sense to consider these steps even in the short term (as part of this PR):

1. Create some base class like SparkStreamingChannel that extends DirectChannel
2. move the configureMessageConverter() method to that new class (no longer static, and only requiring the single contentType parameter)
3. have SparkMessageSender extend SparkStreamingChannel instead of DirectChannel
4. have MessageStoringChannel extend SparkStreamingChannel instead of DirectChannel
5. replace the 2 current calls to this static configureMessageConverter to then call on the MessageStoringChannel and MessageBusSender instances instead